site/docs/v1/tech/installation-guide/cloud.adoc

```diff
@@ -445,6 +448,64 @@ For high availability deployments with purchased support:
 * The FusionAuth engineering team will consult on product implementation and can offer "best practices" advice to further ensure system stability.
 * We size the instance properly for expected traffic, in consultation with the customer.
 
+== Deployment IP Addresses
+
+You may need to know the IP addresses of the instances in your FusionAuth Cloud deployment. There are any number of reasons you might need this information, including:
+
+* Adding the IP addresses to the allow list of an email deliverability vendor.
```

Comment:
> email deliverability vendor.

This sounds strange to me.  Is this what SendGrid is called (for instance)?

Reply:
> * Adding the IP addresses to the allow list for your SMTP service
> * Adding the IP address to an allow list to permit a Webhook events to be delivered through an application firewall

Reply:
Knowing we don't have static IPs, we should also recommend - if at all possible, add the hostname to an Allow List instead of an IP address. I would have to assume some more sophisticated firewalls can manage a hostname instead of the IP and just resolve it in the firewall dynamically. Using the IP address directly should really be a last resort. Webhooks and SMTP servers and such should also always have authentication.

Reply:
Modified per feedback.